### PR TITLE
6086: Fix email validation bug

### PR DESCRIPTION
**Pull request by sam-roberts-bjss**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Organisations.Api/Validators/ApplicationUserValidator.cs [hunk 1]
```diff
@@ -16,6 +16,8 @@ public class ApplicationUserValidator : IApplicationUserValidator
         private const int MaximumLastNameLength = 50;
         private const int MaximumEmailLength = 256;
 
+        private static readonly EmailAddressAttribute _emailAddressAttribute = new EmailAddressAttribute();
```
Comment (assoomro) [inline src/NHSD.BuyingCatalogue.Organisations.Api/Validators/ApplicationUserValidator.cs:19]: This field's name violates our naming rules, ReSharper suggests `EmailAddressAttribute` instead.

Comment (sam-roberts-bjss) [reply src/NHSD.BuyingCatalogue.Organisations.Api/Validators/ApplicationUserValidator.cs:19]: For a static readonly?

Comment (assoomro) [reply src/NHSD.BuyingCatalogue.Organisations.Api/Validators/ApplicationUserValidator.cs:19]: As per our discussion: 
Fault is on my side, my tooling is set to something we haven't agreed on as a team, hence I'm taking back my comment and approving this PR.

Comment (bjss-chris) [reply src/NHSD.BuyingCatalogue.Organisations.Api/Validators/ApplicationUserValidator.cs:19]: I've never been convinced by the default R# setting of suggesting private static fields should be `PascalCase`. I would always go with `camelCase`, which in this code base means they should also have the superfluous leading underscore as @sam-roberts-bjss has done here..

**Pull request merged**